### PR TITLE
Get Date task

### PR DESCRIPTION
**Pull request by danielaSantaCruz**

### CODE REVIEW

File: src/main/java/core/utils/DateManager.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Converts to date a given string with some date.
+ */
+public class DateManager {
+
+    private Calendar today;
```
Comment (edsonarios) [inline src/main/java/core/utils/DateManager.java:15]: The name "today" es very specific, the name should be more general

File: src/main/java/core/utils/DateManager.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+     * Finds the date for one word date.
+     *
+     * @param date a string with the date to find.
+     * @return a date.
+     */
+    public Date oneDateGroup(String date) {
+        today = Calendar.getInstance();
+        if (date.equals("TODAY")) {
+            return addFormatDate(today);
+
```
Comment (edsonarios) [inline src/main/java/core/utils/DateManager.java]: This line should be erase

File: src/main/java/core/utils/DateManager.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        }
+        if (date.equals("TOMORROW")) {
+            today.add(Calendar.DAY_OF_MONTH, 1);
+            return addFormatDate(today);
+        }
+        if (date.equals("YESTERDAY")) {
+            today.add(Calendar.DAY_OF_MONTH, -1);
+            return addFormatDate(today);
+        }
+
```
Comment (edsonarios) [inline src/main/java/core/utils/DateManager.java]: This line should be erase

File: src/test/java/unitests/core/utils/DateManagerTests.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+public class DateManagerTests {
+    @Test
+    public void testGetTodayDate() throws ParseException {
+        DateManager dates = new DateManager();
+        DateFormat parser = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+        String value = "TODAY";
+        Date dateResult = dates.getDate(value);
+        Calendar calendar = Calendar.getInstance();
+        Date date = parser.parse(new SimpleDateFormat("MM/dd/yyyy HH:mm:ss").format(calendar.getTime()));
+        System.out.println(dateResult);
```
Comment (edsonarios) [inline src/test/java/unitests/core/utils/DateManagerTests.java]: This print it's not necessary in a Test

File: src/test/java/unitests/core/utils/DateManagerTests.java [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    @Test
+    public void testGetYesterdayDate() throws ParseException {
+        DateManager dates = new DateManager();
+        DateFormat parser = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss");
+        String value = "YESTERDAY";
+        Calendar calendar = Calendar.getInstance();
+        Date dateResult = dates.getDate(value);
+        calendar.add(Calendar.DAY_OF_MONTH, -1);
+        Date date = parser.parse(new SimpleDateFormat("MM/dd/yyyy HH:mm:ss").format(calendar.getTime()));
+        System.out.println(dateResult);
```
Comment (edsonarios) [inline src/test/java/unitests/core/utils/DateManagerTests.java]: This print it's not necessary in a Test and the rest of Test

File: src/main/java/core/utils/DateManager.java [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+        return null;
```
Comment (GustavoZHA) [inline src/main/java/core/utils/DateManager.java]: Handle null as exception

**Pull request merged**